gcloud-java-dns/src/main/java/com/google/cloud/dns/spi/DefaultDnsRpc.java

```diff
@@ -331,15 +331,14 @@ public Change getChangeRequest(String zoneName, String changeRequestId, Map<Opti
       return getChangeRequestCall(zoneName, changeRequestId, options).execute();
     } catch (IOException ex) {
       DnsException serviceException = translate(ex, true);
-      if (serviceException.code() == HTTP_NOT_FOUND) {
-        if ("entity.parameters.changeId".equals(serviceException.location())
-            || (serviceException.getMessage() != null
-            && serviceException.getMessage().contains("parameters.changeId"))) {
-          // the change id was not found, but the zone exists
-          return null;
-        }
-        // the zone does not exist, so throw an exception
+      if (serviceException.code() == HTTP_NOT_FOUND
```

Comment:
Honestly, I am not sure if that makes the code more readable in this case.
Fine as is if you think otherwise.